app/src/main/java/ch/epfl/sweng/SDP/utils/TypefaceLibrary.java

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.sweng.SDP.utils;
+
+import android.content.Context;
+import android.graphics.Typeface;
+
+/**
+ * Utility class that stores the different typefaces.
+ */
+public class TypefaceLibrary {
+    private static Typeface typeMuro;
+    private static Typeface typeOptimus;
+
+    private TypefaceLibrary() {
+    }
+
+    /**
+     * Instantiates the typefaces given the context.
+     *
+     * @param context the given context
+     */
+    public static void setContext(Context context) {
+        typeMuro = Typeface.createFromAsset(context.getAssets(), "fonts/Muro.otf");
```

Comment:
The two fields are unique per context, not for the whole app right? Otherwise one could define them as final too, by moving the Typeface.createFromAsset inside the constructor and calling here just the constructor.